zaptest/timeout.go

```diff
@@ -21,31 +21,17 @@
 package zaptest
 
 import (
-	"log"
-	"os"
-	"strconv"
 	"time"
-)
 
-var _timeoutScale = 1.0
+	"go.uber.org/zap/internal/ztest"
+)
 
 // Timeout scales the provided duration by $TEST_TIMEOUT_SCALE.
```

Comment:
can you mark this as deprecated? I don't think these functions are intended for end-users right? This should purely be for internal testing

Reply:
Yep, can do.